test/region_growing.jl

```diff
@@ -0,0 +1,128 @@
+get_wrapped{CT<:Colorant}(p::CT) = Images.accum(typeof(p))(p)
```

Comment:
This is the same as `get_wrapped(p::Colorant)`. The name isn't entirely clear, perhaps `of_accum_type` (sort of like `oftype`).